examples/aws_efa/README.md

```diff
@@ -193,6 +193,28 @@ sky launch -c efa efa_vm.yaml
 
 Check the [`efa_vm.yaml`](https://github.com/skypilot-org/skypilot/blob/master/examples/aws_efa/efa_vm.yaml) for the complete SkyPilot cluster yaml configurations.
 
+> **NOTE:** For VMs that support multiple EFA interfaces (e.g., `p4d.24xlarge` with 4 EFA interfaces), you must configure SkyPilot to use internal IPs to access all EFA interfaces.
+>
+> Add the following configuration to `~/.sky/config.yaml`:
+>
+> **Option 1: Using SSH proxy**
+> ```yaml
+> aws:
+>   use_internal_ips: true
+>   vpc_name: <vpc name>
```

Comment:
Is this `vpc_name` necessary? It seems most of the case it should work with the default VPC?